.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,24 @@
+name: Update-docs
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'README.md'
+jobs:
+  transfer-file:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Pushes readme file
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_TOKEN }}
+        with:
+          source_file: 'README.md'
+          destination_repo: 'sourcefuse/arc-docs'
+          destination_folder: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-db'
+          user_email: 'saurabh.jindal@gmail.com'
```

Comment:
Can this be updated to a service account or the author of the PR's username and email?